paper-autocomplete-suggestions.html

```diff
@@ -171,7 +171,7 @@
       <!-- Default suggestion template -->
       <template id="defaultTemplate">
         <paper-item id$="[[_getSuggestionId(index)]]" role="option" aria-selected="false" on-tap="_onSelect">
-          <div>[[_getItemText(item)]]</div>
+          <div inner-h-t-m-l="[[item.text]]"></div>
```

Comment:
I never see this before, can you explain this `inner-h-t-m-l` thingy? thanks

Reply:
Well it binds to `innerHTML` to allow HTML injection.
But this support requires some other changes as well, so hold this PR off for now, will see when I have the time to fix it.